packages/blockchain-wallet-v4/src/transactions/ethereum.js

```diff
@@ -39,17 +39,19 @@ export const getLabel = (address, state) => {
 
 export const _transformTx = curry((addresses, latestBlock, state, tx) => {
   const fee = getFee(tx)
+  const type = toLower(getType(tx, addresses))
```

Comment:
Why use toLower here when you could simply return 'transferred', 'sent', 'received' or 'unknown' all lowercase in the getType function above